shared/src/packages/meditation/libs/validation-schemas/create-meditation-entry-request.validation-schema.ts

```diff
@@ -0,0 +1,22 @@
+import joi from 'joi';
+
+import { ContentType } from '#libs/enums/enums.js';
+
+import { MeditationEntryValidationMessage } from '../enums/enums.js';
+import { type MeditationEntryCreateRequestDto } from '../types/types.js';
+
+const createMeditationEntryRequest = joi.object<
+  MeditationEntryCreateRequestDto,
+  true
+>({
+  mediaUrl: joi.string().trim().required().messages({
+    'any.required': MeditationEntryValidationMessage.MEDIA_URL_REQUIRED,
+    'string.required': MeditationEntryValidationMessage.MEDIA_URL_REQUIRED,
```

Comment:
if I am not mistaken joi doesn't have `string.required`
```suggestion
```

Reply:
Done